content/sensu-core/0.29/quick-start/five-minute-install.md

```diff
@@ -10,7 +10,6 @@ menu:
     parent: quick-start
 ---
 # The Five Minute Install
-{{< platformDropdown "Ubuntu/Debian" "Sensu-Core" "0.29" "Client Installation" "sensu-core">}}
```

Comment:
For posterity, we're removing instances of `platformDropdown` for now because we don't yet have content for alternate platforms.

Reply:
That should be removed in the PR